packages/rx_bloc_cli/mason_templates/bricks/lib_social_logins/__brick__/lib/lib_social_logins/blocs/social_login_bloc.dart

```diff
@@ -47,7 +52,12 @@ class SocialLoginBloc extends $SocialLoginBloc {
   ConnectableStream<bool> _mapToLoggedInState() => _$loginEvent
       .throttleTime(const Duration(seconds: 1))
       .switchMap(
-        (_) => _socialLoginService.login().then((_) => true).asResultStream(),
+        (_) => _socialLoginService.login().then((_) => true).catchError((e) {
+          if (e.toString() == appleCanceled ||
```

Comment:
Please use the error is CanceledErrorModel instead of string comparison to ensure stability of the flow

Reply:
Bonus comment: this error should be thrown from the repository